doc/source/user/configoptions.rst

```diff
@@ -991,8 +991,8 @@ PROJ options
 
       Used by :cpp:func:`OGRLineString::transform`.
 
-      Can be set to YES to remove points that can't be reprojected. See #3758 for the
-      purpose of this option.
+      Can be set to YES to remove points that can't be reprojected. See
+      http://trac.osgeo.org/gdal/ticket/3758 for the purpose of this option.
```

Comment:
```suggestion
      Can be set to YES to remove points that cannot be reprojected. This can for example help reproject lines that have an extremity at a pole, when the reprojection does not support coordinates at poles.
```